examples/medplum-patient-intake-demo/src/pages/UploadDataPage.tsx

```diff
@@ -72,10 +64,114 @@ export function UploadDataPage(): JSX.Element {
   );
 }
 
-async function uploadData(medplum: MedplumClient, data: Bundle): Promise<void> {
-  const result = await medplum.executeBatch(data);
+async function uploadCoreData(medplum: MedplumClient): Promise<void> {
+  const batch = coreData as Bundle;
+
+  const result = await medplum.executeBatch(batch);
+
+  if (result.entry?.every((entry) => entry.response?.outcome && isOk(entry.response?.outcome))) {
+    await setTimeout(
+      () =>
+        showNotification({
+          icon: <IconCircleCheck />,
+          title: 'Success',
+          message: 'Uploaded Core Data',
+        }),
+      1000
+    );
+  } else {
+    throw new Error('Error uploading core data');
+  }
+}
+
+async function uploadExampleData(medplum: MedplumClient): Promise<void> {
+  const exampleDataBatch = exampleData as Bundle;
+  const result = await medplum.executeBatch(exampleDataBatch);
+
+  if (result.entry?.every((entry) => entry.response?.outcome && isOk(entry.response?.outcome))) {
+    await setTimeout(
+      () =>
+        showNotification({
+          icon: <IconCircleCheck />,
+          title: 'Success',
+          message: 'Uploaded Example Data',
+        }),
+      1000
+    );
+  } else {
+    throw new Error('Error uploading example data');
+  }
+}
+
+async function uploadExampleBots(medplum: MedplumClient, profile: Practitioner): Promise<void> {
+  const questionnaires = await medplum.searchResources('Questionnaire');
```

Comment:
@filipeximenes we should be able to use the questionnaire context we set up in App.tsx